source/_components/cover.mqtt.markdown

```diff
@@ -13,39 +13,44 @@ ha_iot_class: "depends"
 
 To use your MQTT cover in your installation, add the following to your `configuration.yaml` file:
 
 ```yaml
-# Example configuration.yml entry
+# Example configuration.yaml entry
 cover:
   - platform: mqtt
     name: "MQTT Cover"
-    state_topic: "home-assistant/cover"
     command_topic: "home-assistant/cover/set"
 ```
 
 Configuration variables:
 
 - **name** (*Optional*): The name of the sensor. Default is `MQTT Cover`.
-- **state_topic** (*Optional*): The MQTT topic subscribed to receive sensor values.
-- **command_topic** (*Optional*): The MQTT topic to publish commands to control the rollershutter.
+- **command_topic** (*Optional*): The MQTT topic to publish commands to control the cover.
 - **payload_open** (*Optional*): The payload that opens the cover. Default is `OPEN`.
 - **payload_close** (*Optional*): The payload that closes the cover. Default is `CLOSE`.
-- **payload_stop** (*Optional*):  The payload that stops the rollershutter. default is `STOP`.
-- **state_open** (*Optional*): The payload that represents open state. Default is `open`.
-- **state_closed** (*Optional*): The payload that represents closed state. Default is `closed`.
+- **payload_stop** (*Optional*):  The payload that stops the cover. default is `STOP`.
+- **state_topic** (*Optional*): The MQTT topic subscribed to receive cover state messages.
+- **state_open** (*Optional*): The payload that represents the open state. Default is `open`.
+- **state_closed** (*Optional*): The payload that represents the closed state. Default is `closed`.
+- **availability_topic** (*Optional*): The MQTT topic subscribed to to receive birth and LWT messages from the MQTT cover device. If `availability_topic` is not defined, the cover availability state will always be "available". If `availability_topic` is defined, the cover availability state will be "unavailable" by default.
+- **payload_available** (*Optional*): The payload that represents the online state. Default is `online`.
+- **payload_not_available** (*Optional*): The payload that represents the offline state. Default is `offline`.
 - **optimistic** (*Optional*): Flag that defines if switch works in optimistic mode. Default is `true` if no state topic defined, else `false`.
-- **qos** (*Optional*): The maximum QoS level of the state topic. Default is `0`. Will also be used when publishing messages.
-- **retain** (*Optional*): If the published message should have the retain flag on or not. Default is `false`.
+- **qos** (*Optional*): The maximum QoS level to be used when receiving and publishing messages.
```

Comment:
I think we should keep the info on default qos value.

Reply:
Agreed. Added.